api/api.py

```diff
@@ -1,7 +1,105 @@
-from flask import Blueprint
+import requests
+import os
+import json
+import sqlite3 as sql
+import config
 
-api_bp = Blueprint('api_bp', __name__)
+from flask import Blueprint, request, jsonify
+from dotenv import load_dotenv
+from stocks.stocks import get_stock_quote, calculate_percent_change, get_stock_name, insert_stock_tracker, get_tracked_stocks_details
 
-@api_bp.route('/')
-def index():
-    return "This is an example api app"
\ No newline at end of file
+load_dotenv()
+
+api_bp = Blueprint("api_bp", __name__)
+
+@api_bp.route("/")
```

Comment:
I don’t think healthcheck should be on the index path, something like /healthcheck should make more sense